src/PanelTraits/Reorder.php

```diff
@@ -41,18 +49,26 @@ public function updateTreeOrder($request)
      * Enable the Reorder functionality in the CRUD Panel for users that have the been given access to 'reorder' using:
      * $this->crud->allowAccess('reorder');.
      *
-     * @param  [string] Column name that will be shown on the labels.
-     * @param  [integer] Maximum hierarchy level to which the elements can be nested (1 = no nesting, just reordering).
+     * If using simple reordering, the field 'seq' is required,
+     * otherwise the reordering required the fields 'parent_id', 'depth', 'lft' and 'rgt'.
+     *
+     * @param  string  $label      Column name that will be shown on the labels.
+     * @param  int     $max_level  Maximum hierarchy level for nesting of entities (1 = no nesting, just reordering).
+     * @param  bool    $simple     Whether reordering should be simple. Will set max level to 1.
+     * @return void
      */
-    public function enableReorder($label = 'name', $max_level = 1)
+    public function enableReorder($label = 'name', $max_level = 1, $simple = false)
```

Comment:
So you can do:

```
->enableReorder('some_name', 500, true)
```

...and `$this->reorder_max_level` turns into the value `(int) 1` which is somewhat evil.

Reply:
Yes, but it's mentioned in the docs, and only done if you specify simple as `true`.

Reply:
Yet, the only way to get `simple` is to set `$max_level` to "something" or `null` and this seems...not so *simple*.

Reply:
I know... it's purely not to mess with backwards compatibility.

Reply:
I had a flash of **brilliance**:

```
if ($max_level === true) {
  $simple = true;
}
```

Reply:
It would “die” if one added `int` type hinting, but otherwise I’d be more inclined to do
```
if ($max_level === false) {
    $max_level = 1;
    $simple = true;
}
```
Since you sort of specify you don’t want a max level = no level… at least in my head